### PR TITLE
K1J-632: Added banner for inactive certificate types.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

**Comment by ChrilleSigerhed** [comment]
image

File: apps/webcert/src/page/CertificatePage.tsx [hunk 1]
```diff
@@ -82,6 +83,7 @@ const CertificatePage: React.FC = () => {
         !isCertificateDeleted && (
           <>
             <MajorVersionNotification />
+            <InactiveCertificateTypeNotification />
```
Comment (mww-sopra-steria) [inline apps/webcert/src/page/CertificatePage.tsx:86]: The comment is not on the correct line, but I add it here as MajorVersionNotification is added here.

Could we add logic to MajorVersionNotification that it is only displayed if inactiveCertificateType not true?

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
See comment. It will remove the risk of it displaying multiple notifications. The certificate could be of a version that is not the latest major of the type and it could happen that the type has been inactivated.

**Review by mww-sopra-steria — APPROVED** [review]
Nice!

**Pull request merged**